mentii/mentii
# Class list component

Makes all of the items in any of the lists of classes a reusable component.

**Significant Changes**
1. Creation of classListItem.component.ts and classListItem.html
2. Moved join class logic into the class list item component
3. Fixed some minor language issues that I came across

**How to Test**
1. Log in
2. Look at the taught list, regular class list (as student) and the public class list
3. Confirm that all look the same
4. Try to join a class in the public class list

**Notes**
Have a wonderful day
Also, haven't added the version number yet. Gonna wait to see what happens with the 3 pull requests ahead of me
